src/main/java/com/bynder/sdk/query/MetapropertyQuery.java

```diff
@@ -44,4 +56,21 @@ public MetapropertyQuery setType(final MediaType type) {
         this.type = type;
         return this;
     }
+
+    public Boolean getOptions() {
```

Comment:
@colinmanning In addition to the changes mentioned from https://github.com/Bynder/bynder-java-sdk/pull/109#issuecomment-1735834057, can you also add some unit tests for these new methods being added?

Reply:
done, my integration tests work using setIds() and I introduced a basic unit test modelled on existing unittest MediaQueryTest